frontend/src/ManageOrgUsers/ManageOrgUsers.jsx

```diff
@@ -54,20 +57,34 @@ class ManageOrgUsers extends React.Component {
         toast.success('User role has been updated', { hideProgressBar: true, position: 'top-center' });
         this.setState({ idChangingRole: null });
       })
-      .catch(( { error } ) => {
+      .catch(({ error }) => {
         toast.error(error, { hideProgressBar: true, position: 'top-center' });
         this.setState({ idChangingRole: null });
       });
   };
 
+  archiveOrgUser = (id) => {
+    this.setState({ archivingUser: id });
+
+    organizationUserService
+      .archive(id)
+      .then(() => {
+        toast.success('User has been removed', { hideProgressBar: true, position: 'top-center' });
+        this.setState({ archivingUser: null });
+        this.fetchUsers();
+      })
+      .catch(({ error }) => {
+        toast.error(error, { hideProgressBar: true, position: 'top-center' });
+        this.setState({ archivingUser: null });
+      });
+  };
+
   createUser = () => {
     this.setState({
-      creatingUser: true
```

Comment:
Ran prettier on this file before modifying it. The diff is a lot easier to read with whitespace off.